docs/zh_cn/guide/mount_at_boot.md

```diff
@@ -6,21 +6,31 @@ slug: /mount_juicefs_at_boot_time
 
 # 启动时自动挂载 JuiceFS
 
-这是一篇关于如何在启动时自动挂载 JuiceFS 的指南。
+在确认挂载成功，可以正常使用以后，可以参考本节内容设置开机自动挂载。
 
 ## Linux
 
-拷贝 `juicefs` 为 `/sbin/mount.juicefs`，然后按照下面的格式添加一行到 `/etc/fstab`：
+挂载命令的 `--update-fstab` 参数能直接帮你做好设置：
```

Comment:
Need to prompt users that this is an unreleased feature

Reply:
this pr is to be merged after 1.1 release

Reply:
Still need to prompt the user that this feature depends on a specific version

Reply:
add notes on available version (v1.1.0)